kafka-init/src/main/java/io/strimzi/kafka/init/Main.java

```diff
@@ -15,10 +18,14 @@ public class Main {
 
     public static void main(String[] args) {
 
-        LOGGER.info("Init-kafka {} is starting", Main.class.getPackage().getImplementationVersion());
+        final String strimziVersion = Main.class.getPackage().getImplementationVersion();
+        LOGGER.info("Init-kafka {} is starting", strimziVersion);
         InitWriterConfig config = InitWriterConfig.fromMap(System.getenv());
 
-        KubernetesClient client = new KubernetesClientBuilder().build();
+        final String userAgent = "fabric8-kubernetes-client/" + Version.clientVersion() 
```

Comment:
Why have you chosen the user agent to look like this?
* I guess it creates overhead on every request?
* Fabric8 client version is normally based on the Strimzi version
* If you need to include Fabric8 version, why not include also the version of the HTTP client?

Reply:
Good point. I changed it to just have the Strimzi component name and version. You can determine the Fabric8 and HTTP client version based on that.